app/controllers/taxa/names_controller.rb

```diff
@@ -200,4 +203,13 @@ def parse_classification_controller_params
     @move_to = params[:move_to]
   end
 
-end
+  def limit_names
+    if current_user.newish? && !current_user.is_trusted_user?
+      names_count = Curator.total_objects_curated_by_action_and_user(Activity.add_common_name.id, current_user.id,
+                    [ChangeableObjectType.synonym.id], "count",
+                    DateTime.now.in_time_zone.to_date.beginning_of_day..DateTime.now.in_time_zone.to_date.end_of_day)
```

Comment:
Line is too long. [117/80]<br>Align the parameters of a method call if they span more than one line.